src/main/java/org/apache/sling/distribution/journal/MessagingProvider.java

```diff
@@ -62,11 +62,14 @@ default Closeable createPoller(
      * @param topicName name of the topic
      * @param reset fallback in case no assign is given or the assigned offset not valid
      * @param assign opaque implementation dependent assign string (can be null)
-     * @param properties list of properties to filter the topic
+     * @param filterProperties list of properties used to filter the topic messages.
```

Comment:
I would describe this only in an abstract way like:
List of properties used to filter the topic messages. Only messages that match all filterProperties keys and values will be received.